axelrod/__init__.py

```diff
@@ -1,4 +1,7 @@
 from __future__ import absolute_import
+import sys
+
+on_windows = sys.platform.startswith("win")
```

Comment:
I've always used ```os.name == 'nt' ``` based on answers at http://stackoverflow.com/questions/4553129/when-to-use-os-name-sys-platform-or-platform-system#11674977